graphql/utils/feature-flags.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable import/prefer-default-export */
+
+export const isGlobalHealthGroupImpactEnabled = () =>
```

Comment:
Is the rationale for not using Growthbook that we currently require user info at runtime to initialize Growthbook? If so, we could create a user-agnostic feature-getter (`getFeatureAtBuildTime` or `getFeatureWithoutUser`).

Your approach works here, and no need to change it, but maybe worth adding a comment in this file that it's a one-off situation because it gets a little weird having two feature flag systems.

Reply:
Nope! I think it makes sense to keep it consistent. I'll switch it to a growthbook feature. For some reason I just interpreted "feature flag" very literally and implemented it similarly to how we implemented other feature flags.

Reply:
Here's a branch with my initial attempts. `jtan/tfac-1011-2`

Reply:
Sounds good! If we edit this in the future, it likely makes sense to use async even for models fetching in-memory data.